dist/backdraft.js

```diff
@@ -1178,31 +1189,63 @@ _.extend(Plugin.factory, {
       var table = this;
       var filter = col.filter;
 
-      // bind focus to click event because of unbinding click from thead th when
-      // installing sort interceptors
-      $('input', head).on("click", function () {
+      // need to manually set focus because of how we unbound filtering events from the head
+      $('input', head).on('click', function() {
         this.focus();
       });
-      // update columnManager filter and ajaxUpdate dataTable when input changed
+
+      // update columnManager filter when input changed
       $('input', head).on('change', function () {
+        var filterIcon = $('.DataTables_filter_wrapper span', head);
         if (filter.type === "list") {
           if (this.checked) {
             filter[this.id] = filter[this.id] || [];
             filter[this.id].push(this.value);
+            filterIcon.removeClass("filterInactive");
+            filterIcon.addClass("filterActive");
           }
-          else {
+          else if (filter[this.id]) {
```

Comment:
it's not clear to me in this context what `this` is or what `this.id` is, and seems like too much inline code here with intricate if / else / else if cases.. hoping some of this logic can be extracted and encapsulated in the templates themselves (and maybe each filter type should be its own View).

Either way can you make it more clear in this code what is going on here and what this.id is? (maybe name it a temporary variable that is better named?